ballerina-tests/tests/ssl_mutual_ssl_test.bal

```diff
@@ -107,8 +107,73 @@ http:ClientConfiguration mutualSslClientConf = {
 };
 
 @test:Config {}
-public function testMutualSsl() {
-    http:Client httpClient = checkpanic new("https://localhost:9116", mutualSslClientConf );
+public function testMutualSsl1() {
+    http:Client httpClient = checkpanic new("https://localhost:9116", mutualSslClientConf1);
+    http:Response|error resp = httpClient->get("/helloWorld15/");
```

Comment:
Can't we use client data binding here ?

```suggestion
    string resp = check httpClient->get("/helloWorld15/");
```

Reply:
All the test cases can be improved for the newly introduced features. That is a separate task to follow I guess.

Reply:
The issue is created to track this. https://github.com/ballerina-platform/ballerina-standard-library/issues/1718